### ISSUE TITLE
Parse errors for output

### ISSUE THREAD

**Issue by KamasamaK** [Microsoft/vscode#40051]
Using VS Code 1.18.1

I am often getting parse errors reported under the Problems/Diagnostics view. It is as if it is expecting the output to be validated in a certain format.

![vscode-parse-errors-output](https://user-images.githubusercontent.com/22951521/33854875-31cdb2c2-de91-11e7-85e4-0c047a08e701.png)

The first one is from an extension called [ftp-simple](https://marketplace.visualstudio.com/items?itemName=humy2833.ftp-simple), which is just outputting actions with timestamps. The other three are empty output sections for default items. As you can see, none of them have an associated `Diagnostic.source` either.

**Comment by ramya-rao-a** [comment]
`output:tasks`, `output:extensions`, `output:search` : Are these files? Or are these referring to the different output channels in the output pane?

**Comment by roblourens** [comment]
I was seeing the same errors earlier, coming from an extension, but I never figured out which extension. If you share you extension list here (Help > Report Issue) one might stand out.

**Comment by KamasamaK** [comment]
@ramya-rao-a These are referring to the output channels in the output pane.

@roblourens These are the extensions I have enabled:

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.1.0
path-intellisense|chr|1.4.2
gitignore|cod|0.5.0
vscode-markdownlint|Dav|0.12.0
vscode-eslint|dba|1.4.3
githistory|don|0.2.3
jquerysnippets|don|0.0.1
xml|Dot|1.9.2
gitlens|eam|6.3.0
EditorConfig|Edi|0.11.1
tslint|eg2|1.0.24
vscode-npm-script|eg2|0.3.3
auto-close-tag|for|0.5.5
code-runner|for|0.8.5
ftp-simple|hum|0.5.8
vscode-cflint|Kam|0.2.4
vscode-cfml|Kam|0.2.0
python|ms-|0.8.0
debugger-for-chrome|msj|3.5.0
vscode-code-outline|pat|0.0.12
java|red|0.15.0
vscode-icons|rob|7.19.0
code-spell-checker|str|1.6.0
tmlanguage|Tog|0.8.4
highlight-matching-tag|vin|0.4.4
vscode-java-debug|vsc|0.4.0
vscode-java-pack|vsc|0.2.0
vscode-todo-highlight|way|0.5.11
vscode-import-cost|wix|2.5.1
vscode-open-in-github|ziy|1.3.0
---

**Comment by ramya-rao-a** [comment]
@KamasamaK Its definitely coming from one of those extensions
Unfortunately I dont have any smart way of determining which one is causing it.

@roblourens Do you have any extensions common with the above list?

**Comment by roblourens** [comment]
I don't see any familiar ones. 

Can you try with the Insiders build?

What prompts those errors to appear? Do they show up whenever you open vscode or only after you take some action?

**Comment by KamasamaK** [comment]
By disabling each extension one at a time, I found that it was [TextMate Languages](https://marketplace.visualstudio.com/items?itemName=Togusa09.tmlanguage).

The errors appear after viewing the particular output channel in the output pane.

This happens even when I do not have any files opened that would satisfy any of the `activationEvents`, so I assume something from VS Code is triggering activation. Would VS Code reading settings, for example, trigger `onLanguage:json`? I can see in the console that it is being activated when opening the workspace.

I have reported this issue under the extension as Togusa09/vscode-tmlanguage#14.
